docs/source/runway_config.rst

```diff
@@ -1,7 +1,7 @@
 .. _AWS CDK: https://docs.aws.amazon.com/cdk/latest/guide/getting_started.html
 .. _CloudFormation: https://aws.amazon.com/cloudformation/
 .. _Serverless Framework: https://serverless.com/
-.. _CFNgin: cfngin/index.html
+.. _CFNgin: terminology.html#runway-s-cfngin
```

Comment:
just checked where this is pointed to. the original link was to the CFNgin documentation (what it is, how to configure it, etc). This one just goes to [CFNgin terms](https://docs.onica.com/projects/runway/en/release/terminology.html#runway-s-cfngin) (not even info about what CFNgin is).

Maybe it would be better to point it to [index.html#cloudformation-troposphere](https://docs.onica.com/projects/runway/en/release/index.html#cloudformation-troposphere) to at least provide information about what it is?

Reply:
Updated